monai/fl/client/__init__.py

```diff
@@ -9,5 +9,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .client_algo import ClientAlgo
-from .monai_algo import MonaiAlgo
+from .client import BaseClient, ClientAlgo, ClientStats
```

Comment:
the module name changes will break code written with monai v1.0.0, ideally we maintain compatibility util v1.2...

Reply:
Good point. We will keep the old filenames to allow the previous way of importing.